README.md

```diff
@@ -72,9 +84,8 @@ spaceImport(options)
 
 ## Limitations
 
-- This tool imports only your published Content Model, Content and Assets. It will support Editor Interfaces, Webhooks and Roles & Permissions in a future version. Draft entries are not supported.
-- You can only import content to an empty space. It does not support merging content into pre-existent spaces.
-- If you have custom widgets, you need to reinstall them manually in the new space.
+- This tool currently does **not** support the import of roles & permissions and space memberships.
+- If you have custom UI extensions, you need to reinstall them manually in the new space.
```

Comment:
Can you also mention that the webhooks with credential will be imported as a normal webhook and the credential should be entered manually